Cargo.toml

```diff
@@ -1,9 +1,9 @@
 [workspace]
 members = [
     "musig",
-    "musig_wasm",
+    "wasm",
 ]
 
 [profile.release]
 # Tell `rustc` to optimize for small code size.
-opt-level = "s"
+opt-level = "s"
```

Comment:
Nit: Here an below: please fix missing newline at the end of file.

Most likely, you can set up an editor to add them automatically.